src/popup-page/modules/app.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { getItem, StoreObserver, setItem } from '../../utils/storage'
```

Comment:
Seems to be unused, right?
```suggestion
import { getItem, setItem } from '../../utils/storage'
```